changelog.txt

```diff
@@ -1,5 +1,17 @@
 *** WooCommerce Services Changelog ***
 
+= 1.24.0 - 2020-xx-xx =
+* Fix   - PHP 7.4 notice for taxes at checkout.
+* Add   - Carrier logos next to rates.
+
+= 1.23.2 - 2020-06-12 =
+* Fix   - Refund not possible on order page.
+
+= 1.23.1 - 2020-06-10 =
+* Tweak - Update WooCommerce compatibility to 4.2
+* Fix	- Taxjar broken in admin's new order page
+* Fix   - Tax recalculation on admin order screen missing street address.
+
```

Comment:
I needed to add an entry for this change but the changelog was out-of-date so I copied in all the entries from develop.